crates/analyzer/tests/errors.rs

```diff
@@ -339,7 +333,7 @@ test_file! { unsafe_misuse }
 test_file! { unsafe_nesting }
 
 test_ingot! { bad_ingot }
-test_ingot! { mainless_ingot }
+// test_ingot! { mainless_ingot }
```

Comment:
commented out since the mode check has been moved to the build file resolution phase

new tests should be added for build file errors